modules/adding-workbench-pod-tolerations.adoc

```diff
@@ -41,7 +41,7 @@ endif::[]
 
 .Verification
 
-. In the {openshift-platform} console, for a pod that is running, click *Workloads* -> *Pods*. Otherwise, for a pod that is stopped, click *Workloads* -> *StatefulSet*.
+. In the {openshift-platform} console, click *Workloads* -> *StatefulSet*. You can see how many pods are running, either zero or one, depending on whether your workbench is currently started or stopped. 
```

Comment:
```suggestion
. In the {openshift-platform} console, select namespace (is the name of Data Science Project or you may find it under `rhods-notebooks`) click *Workloads* -> *StatefulSet*. You can see how many pods are running, either zero or one, depending on whether your workbench is currently started or stopped. 
```

Reply:
this is a great point, having the wrong project selected on top left side of that screen is often cause for user confusion

<img width="179" height="122" alt="Screenshot 2025-07-31 at 5 29 44 PM" src="https://github.com/user-attachments/assets/d4cf73a6-3ab5-4146-bd18-890b166c5d82" />